src/ui/server/content-strings/fields/insurance/declarations/index.ts

```diff
@@ -0,0 +1,35 @@
+import DECLARATIONS_FIELDS_IDS from '../../../../constants/field-ids/insurance/declarations';
+
+const { WILL_ADHERE_TO_ALL_REQUIREMENTS, HAS_NO_OFFENSES_OR_INVESTIGATIONS, IS_NOT_AWARE_OF_EXISTING_SLAVERY } = DECLARATIONS_FIELDS_IDS.MODERN_SLAVERY;
+
+export const DECLARATIONS_FIELDS = {
+  MODERN_SLAVERY: {
+    [WILL_ADHERE_TO_ALL_REQUIREMENTS]: {
+      VERSIONS: [
+        {
+          VERSION: '1',
+          LABEL:
+            'Do you adhere to, and intend to continue to adhere to, all requirements that apply to you as set out in the Modern Slavery Act 2015, including reporting obligations, and all applicable legislation relating to the prevention of modern slavery in every country that you operate in?',
+        },
+      ],
+    },
+    [HAS_NO_OFFENSES_OR_INVESTIGATIONS]: {
+      VERSIONS: [
+        {
+          VERSION: '1',
+          LABEL:
+            'Do you confirm you are not currently under investigation and have not been fined, convicted or found guilty of any offences under the Modern Slavery Act 2015, or under any applicable similar laws or regulations relating to the prevention of modern slavery or any similar infringement of human rights in any jurisdiction?',
+        },
+      ],
+    },
+    [IS_NOT_AWARE_OF_EXISTING_SLAVERY]: {
+      VERSIONS: [
+        {
+          VERSION: '1',
+          LABEL:
+            'To the best of your knowledge, can you confirm you are not aware of the existence of Modern Slavery  in respect of yourself, the export contract (s), your immediate holding company or your supply chain, in each case,  at present or in the past 2 years.',
```

Comment:
should there be a space between contract and (s)?

Reply:
This matches the design.

Reply:
might need to double check with design.  Can be fixed later